test/unit/loggingManagerTests.js

```diff
@@ -365,6 +389,103 @@ describe('LoggingManager', function () {
     })
   })
 
+  describe('checkLogLevelMetadata', function () {
+    beforeEach(function () {
+      this.logger = this.LoggingManager.initialize(this.loggerName)
+    })
+
+    describe('checkLogLevel', function() {
+      it('should request log level override from google meta data service', async function() {
+        this.logger.getTracingEndTime = this.logger.getTracingEndTimeMetadata
+        await this.logger.checkLogLevel()
+        const options = {
+          headers: {
+            'Metadata-Flavor': 'Google'
+          } 
+        }
+        const uri = `http://metadata.google.internal/computeMetadata/v1/project/attributes/${this.loggerName}-setLogLevelEndTime`
+        this.Fetch.should.have.been.calledWithMatch(uri,options)
```

Comment:
Unfortunately the prettier-eslint command is malformed, so this was not picked up. Let's pull in https://github.com/overleaf/logger-module/pull/24 and rerun the `format:fix` command.